integrations/data/TestStream_InfluxQLNodeMissingValue.srpl

```diff
@@ -0,0 +1,39 @@
+dbname
+rpname
+packets value=1000 0000000001
+dbname
+rpname
+packets value=1001 0000000002
+dbname
+rpname
+packets value2=0 0000000003
+dbname
+rpname
+packets value=1003 0000000004
+dbname
+rpname
+packets value=1004 0000000005
+dbname
+rpname
+packets value=1006 0000000006
+dbname
+rpname
+packets value=1007 0000000007
+dbname
+rpname
+packets value=1007 0000000008
+dbname
+rpname
+packets value=1008 0000000009
+dbname
+rpname
+packets value=1009 0000000010
+dbname
+rpname
+packets is_missing_value=1011 0000000011
```

Comment:
@nathanielc 
Notice how in the test `TestStream_InfluxQLNodeMissingValue_Stream` the data's timestamp is off by one. I'm not sure if that is intentional behavior or not.

```go
 +	er := models.Result{
 +		Series: models.Rows{
 +			{
 +				Name:    "packets",
 +				Tags:    nil,
 +				Columns: []string{"time", "mean"},
 +				Values: [][]interface{}{[]interface{}{
 +					time.Date(1971, 1, 1, 0, 0, 10, 0, time.UTC),
 +					1011.0,
 +				}},
 +			},
 +		},
 +	}
```